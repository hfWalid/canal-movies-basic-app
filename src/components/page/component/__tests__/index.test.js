import React from 'react';
import { cleanup, render } from 'react-testing-library';
import Page from "../index";

jest.mock('react-router-dom', () => ({
    useLocation: () => ({
        push: jest.fn(),
    }),
}));

// automatically unmount and cleanup DOM after the test is finished.
afterEach(cleanup);

describe('renders page', () => {
    test('renders page', async () => {
        const { asFragment } = render(<Page />,);
        expect(asFragment()).toMatchSnapshot();
    });
});
