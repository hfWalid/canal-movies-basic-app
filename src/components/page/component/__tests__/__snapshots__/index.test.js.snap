// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders page renders page 1`] = `
<DocumentFragment>
  <article>
    <section>
      <h2 />
    </section>
  </article>
</DocumentFragment>
`;
